rockcraft/commands/init.py

```diff
@@ -118,6 +116,30 @@ class InitCommand(AppCommand):
                 #       - flask/app/webapp
                 #       - flask/app/templates
                 #       - flask/app/static
+                #       # You may need packages to build a python package. Add it here if necessary
+                #       build-packages:
+                #           - pkg-config
+                #           - libxmlsec1-dev
+
+                # You can add package slices or Debian packages to the image.
+                # Package slices are subsets of Debian packages, which result
+                # in smaller and more secure images.
+                # See https://documentation.ubuntu.com/rockcraft/en/latest/explanation/chisel/
+
+                # Add this part if you want to add packages slices to your image.
+                # You can find a list of packages slices in https://github.com/canonical/chisel-releases
+                #   flask-framework/runtime-slices:
+                #     plugin: nil
+                #     stage-packages:
+                #     # list required package slices for your flask application below.
+                #     - libxml2_libs
```

Comment:
Should we make it clearer (somehow) that libxml2_libs, and the others, are just examples?

Reply:
Updated, a bit verbose but clearer